akka-docs/src/main/paradox/stream/operators/Source-or-Flow/filter.md

```diff
@@ -29,3 +29,6 @@ it returns false the element is discarded.
 
 @@@
 
+## API docs
+
+@apidoc[Flow.filter](Flow) { scala="#filter(p:Out=%3EBoolean):FlowOps.this.Repr[Out]" java="#filter(akka.japi.function.Predicate)" }
```

Comment:
The java version doesn't seem to work for me: it links to https://doc.akka.io/japi/akka/2.6/?akka/stream/javadsl/Flow.html#filter(akka.japi.function.Predicate) , and the inner summary correctly switches to the `Flow` page but not to that id in the `Flow` page... is there something we can do about that?

I guess if not then it's still OK to link to the right method for scaladoc and to the class for javadoc

Reply:
The frames break it. The anchor is not picked up by the Javadoc page.
We could use direct links without the `?`.
Should we even switch to Javadoc 11 to get rid of the frames altogether?

Reply:
actually I think we're already using jdk11 to generate the javadocs, but still using `--frames`. I guess we can/should remove that as it is 'the future' anyway...